### DISCUSSION TITLE
Multi dev developing

### DISCUSSION THREAD

**Discussion by W4CKYFR** [godotengine/godot-proposals#10119] [Engine Core]
It would be very nice if I could work on games with my friends, work partners or just anyone without having to uses something like GIT to do it. It would be a very nice addition that might even interest bigger gaming companies to godot. That's all, thanks.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/851.
